README.md

```diff
@@ -59,7 +59,7 @@ This file is maintained by velocitas CLI, do not modify manually. Change setting
 
 - OS Recommendation is e.g. Ubuntu >= 22.04
 - python3 (If not default in your environment create a symlink or use `python-is-python3`)
-- `wget`, `dapr`, `build-essential`, `glibc`, `git` need to be installed
+- `wget`, `build-essential`, `glibc`, `git` need to be installed
 
 ## Project configuration
 
```

Comment:
Below (in the paragraph behind the .velocitas.json code snippet) it's stated: "In a future feature addition, we will allow arbitrary git repository URLs in the name field."
Isn't that something, we already support?